lib/runCode.js

```diff
@@ -171,16 +171,19 @@ module.exports = function (opts, cb) {
       args.reverse()
       args.push(runState)
```

Comment:
I find this quite unintuitive now that I think about it. I'd prefer passing an object with named arguments, something like
```
var args = {stack: runState.stack, cb: asyncCb, ...};
...
var result = opFn.apply(null, args);
```
and then use es6 destructuring to name the opcode inputs in opFn.

Not now, just thinking out loud.